packages/image_picker/image_picker_ios/pubspec.yaml

```diff
@@ -26,3 +26,9 @@ dev_dependencies:
     sdk: flutter
   mockito: ^5.0.0
   pigeon: ^3.0.2
+
+
+# FOR TESTING ONLY. DO NOT MERGE.
```

Comment:
You can revert all of these from every package since only one package will need code changes now.